src/dexter/log/context.cr

```diff
@@ -0,0 +1,9 @@
+class Log::Context
+  alias ContextValue = Bool | Float32 | Float64 | Int32 | Int64 | String | Time
+
+  def to_h : Hash(String, ContextValue)
+    as_h.transform_values do |v|
+      v.raw.as?(ContextValue)
+    end.compact
+  end
```

Comment:
The `Log::Context` is extremely awkward to use because the type can be any number of types, many of which I don't think are even possible to add to the context. I'll open an issue about it, but for now I've added a `to_h` that returns the key/value data in a more usable Hash object